BookPlayer/Services/BookSortService+SortError+PlayListSortOrder.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+final class BookSortService {
+
+    var books: NSOrderedSet
+
+    init(books: NSOrderedSet) {
+        self.books = books
+    }
+
+    public func perform(filter type: PlayListSortOrder) throws -> NSOrderedSet {
```

Comment:
I don't think `perform(filter type...` conveys the appropriate meaning, since we're not actually filtering but sorting. we could rename it to `sort(type: PlayListSortOrder...`